build.gradle.kts

```diff
@@ -7,6 +7,8 @@ plugins {
     id("maven-publish")
     id("signing")
     id("com.github.sherter.google-java-format") version "0.8"
+    id( "org.ysb33r.java.modulehelper") version("0.9.0") apply false
+    id("com.github.johnrengelman.shadow") version ("5.2.0") apply false
```

Comment:
I don't understand why these would be included here but not applied and then shadow isn't even applied in subprojects.  If the shading is being removed, shouldn't the use of this plugin also be removed?

Reply:
Ah right, if we forego the shading I'll have to remove the shadow plugin - I missed that.

> I don't understand why these would be included here but not applied

The reason is of maintenance. Declare all of the Gradle plugins with their versions in one place which is the root project. Then only apply them selectively in the subproject that requie them. This approach suffuce for probably 98%+ os use cases. It is rare to reuire two different version of a plugin in the same multiproject.